scripts/backup_repos.sh

```diff
@@ -1,17 +1,21 @@
-cd $1/podverse-api
+#!/bin/bash
+
+[ -z "${1}" ] && exit 1
+
+cd "${1}/podverse-api" || eixt 2
 git pull origin develop
 
-cd $1/podverse-fdroid
+cd "$1/podverse-fdroid" || eixt 2
 git pull origin develop
 
-cd $1/podverse-ops
+cd "$1/podverse-ops" || eixt 2
 git pull origin master
 
-cd $1/podverse-rn
+cd "$1/podverse-rn" || eixt 2
 git pull origin develop
 
-cd $1/podverse-web
+cd "$1/podverse-web" || eixt 2
 git pull origin develop
 
-cd $1/v4v-wallet-browser-extension
+cd "$1/v4v-wallet-browser-extension" || eixt 2
```

Comment:
is this supposed to be "eixt 2"?

Reply:
nope that was a typo, should be exit. thanks copy/paste

on the exit code side, the numbers don't really matter. I was just trying to give a way to see the type
Though looking at https://tldp.org/LDP/abs/html/exitcodes.html code 2 has a bit more definition 'Misuse of shell builtins (according to Bash documentation)'